gaia-rs/src/client.rs

```diff
@@ -58,6 +61,10 @@ pub enum GaiaQueryCommands {
     Staking(StakingQueryCli),
     /// Querying commands for the ibc module
     Ibc(IbcQueryCli),
+    /// Querying tendermint for a transaction
+    Tx(TxQueryCli),
+    /// Querying tendermint for transactions by event
+    Txs(TxsQueryCli),
```

Comment:
Would it be possible to have the new commands added automatically inside gears?

Reply:
moved commands to gears